crates/typst-library/src/math/frac.rs

```diff
@@ -47,11 +50,17 @@ pub struct BinomElem {
 
     /// The binomial's lower index.
     #[required]
-    pub lower: Content,
+    #[variadic]
+    pub lower: Vec<Content>,
 }
 
 impl LayoutMath for BinomElem {
     fn layout_math(&self, ctx: &mut MathContext) -> SourceResult<()> {
+        // Prevents one element binomials
+        if self.lower().is_empty() {
+            bail!(self.span(), "missing argument: lower");
+        }
```

Comment:
Ideally, this check would be performed as part of argument parsing with a `#[parse]` annotation on `pub lower`. See `matrix.rs` for an example of how to do this.

Reply:
I have tried to understand how to correctly pass this argument but have failed. I have installed the GitHubCLI and have run commands like `gh run --update` but to no avail. How is it done?